### PR TITLE
[WIP] Payments app

### PR DESCRIPTION
**Pull request by patrick91**
Implements an app to handle and store payments.

TODO:

- [ ] Payment with stripe 
- [ ] Allow to manually approve payments from the admin
- [ ] Allow to refund payments

To be considered:

- [ ] Payment history, shall we keep track of all the changes of a payment?

### CODE REVIEW

**Comment by patrick91** [comment]
I've added a stub for a Stripe Provider, I need to create the following methods:

- `pay` or `capture`
- `refund`

and potentially some more

**Comment by simobasso** [comment]
@patrick91 please squash fdda1b2 1380715 and d9cae5e together (I suppose this branch becomes big quickly 🌮 ) thanks !

(django-model-utlis > payaments seed > payament admin)

File: backend/payments/models.py [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+        ('refunded', _('refunded')),
+        ('completed', _('completed')),
+        ('error', _('error')),
+    )
+
+    PROVIDERS = Choices(
+        ('stripe', _('stripe')),
+        ('bank_transfer', _('bank transfer')),
+    )
+
```
Comment (simobasso) [inline backend/payments/models.py]: missing `user` FK

**Comment by simobasso** [comment]
Hi @astagi thanks for your pr!

I have to ask you to fix the conflict and change the git history:

- the app seed, patrick implementation, your implementation, 1 migration
- app implementation and the typo

thanks for your contribution, feel free to ask :)

**Comment by patrick91** [comment]
@astagi friendly ping :)

**Pull request closed without merging**